src/test/java/org/openstreetmap/atlas/checks/validation/points/LoneNodeCheckTest.java

```diff
@@ -0,0 +1,44 @@
+package org.openstreetmap.atlas.checks.validation.points;
+
+import org.junit.Assert;
+import org.junit.Rule;
+import org.junit.Test;
+import org.openstreetmap.atlas.checks.configuration.ConfigurationResolver;
+import org.openstreetmap.atlas.checks.validation.verifier.ConsumerBasedExpectedCheckVerifier;
+
+/**
+ * {@link LoneNodeCheck} unit tests.
+ * 
+ * @author mm-ciub on 06/04/2021.
+ */
+public class LoneNodeCheckTest
```

Comment:
It would be great to have a unit tests to check for connections to highway edges and to test the configurable.